### DISCUSSION TITLE
aws configure import --csv doesn't work

### DISCUSSION THREAD

**Discussion by VladyslavZagorodnyi** [aws/aws-cli#8129] [Q&A] [answered]
I'm trying to authenticate via aws cli (https://awscli.amazonaws.com/v2/documentation/api/latest/reference/configure/import.html) on the instance, I'm using credentials located at csv file (I added to csv file the next values:
- User name,
- Access key ID,
- Secret access key. 

So actual command is:

> aws configure import --csv file://credentials.csv

I get such output:

> Successfully imported 1 profile(s)

After I want to see running clusters at the aws account I'm logged in, so I run the next command: 

> aws eks list-clusters --region=us-east-2 

but I get such output:

> An error occurred (UnrecognizedClientException) when calling the ListClusters operation: The security token included in the request is invalid. ,

 so it looks like I'm not authenticated.
 
 @bisdavid , @jamesls , @awstools , @garnaat , @JordonPhillips , @nateprewitt , @AWS-SF - maybe somebody faced such case, pls help.

**Comment by RyanFitzSimmonsAK** [comment]
HI @VladyslavZagorodnyi, thanks for reaching out.  After importing your credentials, could you run `aws sts get-caller-identity` and ensure that the correct profile is being used?  If not, try adding `--profile Username` to the eks command, obviously replacing Username with whatever your profile is named.

**Reply by VladyslavZagorodnyi** [reply]
Hi @RyanFitzSimmonsAK , I tried but it doesn't work , see outputs:
![Screenshot 2023-08-24 at 14 06 22](https://github.com/aws/aws-cli/assets/29700638/0d80ebab-c595-4872-9461-5dd14be81b84)

I tried --profile Username , as well as --profile=Username

**Reply by RyanFitzSimmonsAK** [reply, marked as answer]
Try clearing your credentials file of any existing profiles, and clearing any existing environment variables.  If that doesn't work, generate a new set of credentials and try again.

@github-actions proposed answer
